lib/Doctrine/DBAL/Exception/InvalidPlatformType.php

```diff
@@ -30,7 +30,7 @@ public static function new($invalidPlatform) : self
 
         return new self(
             sprintf(
-                "Option 'platform' must be an object and subtype of '%s'. Got '%s'",
+                'Option "platform" must be an object and subtype of "%s". Got "%s".',
```

Comment:
Since the class is a symbol, the quotes are not needed. Got "%s" should use the variable formatter. Otherwise, it will print Got "int" for an integer which is misleading.